HerPublicWebsite.BusinessLogic/Models/ReferralRequest.cs

```diff
@@ -61,6 +62,8 @@ public ReferralRequest(Questionnaire questionnaire)
         ContactEmailAddress = questionnaire.LaContactEmailAddress;
         ContactTelephone = questionnaire.LaContactTelephone;
         RequestDate = DateTime.Now;
```

Comment:
Now that we have both `RequestDate` and `PendingLocalAuthorityWhenSubmitted` behaving slightly differently from the rest of the fields here, I think that it would help readability if you add a line break before `RequestDate`.

```
    IncomeBand = questionnaire.IncomeBand!.Value;
    FullName = questionnaire.LaContactName;
    ContactEmailAddress = questionnaire.LaContactEmailAddress;
    ContactTelephone = questionnaire.LaContactTelephone;

    RequestDate = DateTime.Now;
    PendingLocalAuthorityWhenSubmitted =
        questionnaire.LocalAuthorityHug2Status == LocalAuthorityData.Hug2Status.Pending;
}
```